docusaurus/docs/React/components/contexts/component-context.mdx

```diff
@@ -256,12 +256,14 @@ Custom UI component to display system messages.
 
 ### MessageTimestamp
 
-Custom UI component to display a timestamp on a message.
+Custom UI component to display a timestamp on a message. This does not include a timestamp for edited messages.
```

Comment:
Here it would be nice to include a link to `MessageEditedTimestamp` link if it was overridable.

Reply:
I'll add a link to `Timestamp` for now.